crates/telio-wg/src/wg.rs

```diff
@@ -360,7 +428,7 @@ impl WireGuard for DynamicWg {
         Ok(task_exec!(&self.task, async move |s| {
             let mut to = s.interface.clone();
             to.peers.remove(&key);
-            s.update(&to, true).await;
+            let _ = s.update(&to, true).await;
```

Comment:
Just a tip for the future: unrelated changes could go into a separate commit, no need to squash everything as it inflates the commit's value :)